### DISCUSSION TITLE
How to jump back to the last position in the book?

### DISCUSSION THREAD

**Discussion by netsonicyxf** [sumatrapdfreader/sumatrapdf#2721] [Documentation]
When reading PDF or Epub, etc, if I click a link, eg. an endnote, it will jump to another location, 
then how can I jump back to the last position in the book?

**Comment by sergsako** [comment]
According to [**Manual**](https://www.sumatrapdfreader.org/docs/Keyboard-shortcuts), you can use **Alt + Left Arrow** keys. It works. According to **Menu / Go To** , you can use **Backspace** key. It also works. Or simply use **Menu / Go To / Back** (a long way).
